jenkins/ci.opensuse.org/openstack-trackupstream.yaml

```diff
@@ -150,13 +150,6 @@
           export OBS_CHECKOUT=$JHOME/OBS_CHECKOUT/$OBS_PROJECT
           export OSC_BUILD_ROOT=$JHOME/buildroot
 
-          mkdir -p ~/.obs
-          for i in incoming repo repourl
-          do
-            mkdir -p $JHOME/obscache/tar_scm/$i
-          done
```

Comment:
This is not part of the trackupstream script. So either keep it here or move it to the script as well.